app/assets/javascripts/admin/addon/components/site-setting.js

```diff
@@ -1,18 +1,20 @@
+import { readOnly } from "@ember/object/computed";
 import BufferedContent from "discourse/mixins/buffered-content";
 import Component from "@ember/component";
 import SettingComponent from "admin/mixins/setting-component";
 import SiteSetting from "admin/models/site-setting";
-import { readOnly } from "@ember/object/computed";
 
-export default Component.extend(BufferedContent, SettingComponent, {
-  updateExistingUsers: null,
+export default class SiteSettingComponent extends Component.extend(
+  BufferedContent,
+  SettingComponent
+) {
+  updateExistingUsers = null;
 
+  @readOnly("setting.staffLogFilter") staffLogFilter;
```

Comment:
```suggestion
  @readOnly("setting.staffLogFilter") staffLogFilter;

```

For a second there I thought `_save()` is `@readOnly` 😛